tests/e2e/specs/App.spec.ts

```diff
@@ -0,0 +1,40 @@
+import Show from '@/models/Show';
+
+describe('App.vue', () => {
+  it('Menus and grapher exists', () => {
```

Comment:
fine for now, but end to end tests should be more centered around user stories. this test as it is currently is more of a snapshot test (could look into visual regression tests if you want)